evm-contracts/src/v0.6/tests/MedianTestHelper.sol

```diff
@@ -1,15 +1,17 @@
 pragma solidity ^0.6.0;
 
-import "../Median.sol";
+import '../Median.sol';
 
 contract MedianTestHelper {
+    function publicGet(int256[] memory list) public pure returns (int256) {
```

Comment:
This file also doesn't seem to be linted since it should be 2 spaces instead of 4.